internal/status/routeconditions.go

```diff
@@ -28,6 +28,7 @@ const ResourceTLSRoute = "tlsroutes"
 
 const ConditionNotImplemented gatewayapi_v1alpha1.RouteConditionType = "NotImplemented"
 const ConditionResolvedRefs gatewayapi_v1alpha1.RouteConditionType = "ResolvedRefs"
+const ConditionValidForwardTos gatewayapi_v1alpha1.RouteConditionType = "ValidForwardTos"
```

Comment:
Note that this condition type and reason are not defined by Gateway API. Feel free to suggest alternatives if preferred.

Reply:
seems reasonable to me to add our own conditiontype/reason, don't see any upstream

seems like this reasonably captures what we're trying to convey

Reply:
Yeah, I don't think that the ResolvedRefs condition really matches this either, so we should do our own for now. Off the top of my head, I can't think of a better name though. I will also make a note to raise this upstream and see if we can put something into upstream.

Reply:
We still need to set status on the Gateway with a resolved refs, but that can be a different issue.

Reply:
We should also somehow build out some docs on this to explain how Contour will work since Gateway doesn't have any docs on how this works.